assets/js/modules/adsense/index.js

```diff
@@ -78,6 +78,7 @@ export const registerModule = ( modules ) => {
 					data: null,
 				};
 			},
+			screenWidgetContext: CONTEXT_MODULE_ADSENSE,
 		}
 	);
 };
```

Comment:
This file is missing the logic to register the AdSense main widget area (it's only missing here; you've added it as needed in Analytics and Search Console).

Reply:
Good catch!